docs/operate/reference/components/motor/dmc4000.md

```diff
@@ -9,8 +9,14 @@ aliases:
   - "/components/motor/dmc4000/"
 component_description: "Stepper motor driven by a DMC-40x0 series motion controller."
 # SMEs: Rand, James
+draft: true
```

Comment:
If you put this in draft mode then the site won't generate. Did you just want to hide this?